data/sql/updates/pending_db_world/omroggequip.sql

```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `creature_equip_template` WHERE `CreatureID` = 16809 AND `ID` IN (1, 2);
+INSERT INTO `creature_equip_template` (`CreatureID`, `ID`, `ItemID1`, `ItemID2`, `ItemID3`, `VerifiedBuild`)
+VALUES 
+(16809, 1, 29484, 0, 0, 18019),
+(16809, 2, 29479, 0, 0, 18019);
```

Comment:
VerifiedBuild is the exact build (Retail) from which the value was taken, since I took it from build 48999, we'll go with that. 18019 is MoP afaik.
Also missing newline at the end of file
```suggestion
DELETE FROM `creature_equip_template` WHERE `CreatureID` = 16809 AND `ID` IN (1, 2);
INSERT INTO `creature_equip_template` (`CreatureID`, `ID`, `ItemID1`, `ItemID2`, `ItemID3`, `VerifiedBuild`) VALUES 
(16809, 1, 29484, 0, 0, 48999),
(16809, 2, 29479, 0, 0, 48999);

```